TopPlaces/TopPlaces/Controllers/DetailsPhotoViewController.m

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+
+#import "DetailsPhotoViewController.h"
+
+#import "PhotosHistory.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DetailsPhotoViewController()<UIScrollViewDelegate>
+
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *spinner;
+@property (weak, nonatomic) IBOutlet UIScrollView *scrollView;
+@property (weak, nonatomic) IBOutlet UILabel *noImageLoadedView;
+@property (strong, nonatomic) UIImageView *imageView;
+
+@end
+
+@implementation DetailsPhotoViewController
+#pragma mark - UIView overrides
```

Comment:
Spacing.
Also we usually prefer :
#pragma mark -
#pragma mark Some Title
#pragma mark -

This is much more apparent when one reads the code.